Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04
+
+#Update ...
+RUN apt-get update
+RUN apt-get upgrade -y
```

Comment:
Is it really necessary ?

Reply:
To have the latest version of everything in the bot because of its interaction with Telegram which is continuously updated ?

Reply:
Isn't that the job of the docker image ([https://hub.docker.com/_/ubuntu](https://hub.docker.com/_/ubuntu)), e.g. with `docker pull ubuntu:20.04`?

Reply:
you right ! they are useless.